### ISSUE TITLE
Can't be used in ViewPager because swipe doesn't work

### ISSUE THREAD

**Issue by anthonybattel** [jsibbold/zoomage#83]
Unfortunately it doesn't work with ViewPager. I can zoom in the photo but the ViewPager's swipe is broken so I can't see my next photos in the slideshow.
[This library](https://github.com/chrisbanes/PhotoView)  seems to work with but there is no customization like yours

**Comment by DanGdl** [comment]
Try to override onTouch behavior: swipe in normal scale - don't handle it (so view pager can get this event)

**Comment by jsibbold** [comment]
working on getting a fix for this into a snapshot soon

**Comment by jsibbold** [comment]
snapshot is up for this, 1.3.1-SNAPSHOT

**Comment by anthonybattel** [comment]
Thanks a lot

**Comment by jsibbold** [comment]
No problem, if you run into issues with it, let me know, otherwise I'll look to get this out of snapshot and into official release in a couple weeks.

**Comment by usmanrana07** [comment]
@anthonybattel can you please share the results if you have used this update?

**Comment by jsibbold** [comment]
1.3.1 is in jcenter now

**Comment by anthonybattel** [comment]
@usmanrana07 I finally used [this library](https://github.com/chrisbanes/PhotoView) because 1.3.1 zoomage version was not released . You can handle some events this way in Kotlin :

`photo_view.apply {`
        `       setOnScaleChangeListener { _, _, _ ->`
        `           doSomething()`
        `}`
        `setOnPhotoTapListener { _, _, _ ->`
        `doSomething()`
        `}`
        `setOnOutsidePhotoTapListener {`
        `doSomething()`
        `}`
        `}`

I have not tested 1.3.1 zoomage

**Comment by jsibbold** [comment]
Are you still seeing issues scrolling or swiping with 1.3.1?

**Comment by usmanrana07** [comment]
If I compare with PhotoView then Zoomage is missing less touches during swipe in a vertical ViewPager2 (means performing better in swipe). But still at some images with need to swipe twice/thrice. Another thing is that we don't have support to override LongPress method.

**Comment by usmanrana07** [comment]
Dragging the zoomed image with single finger is not smooth.

**Comment by jsibbold** [comment]
You're having to swipe multiple times with only some images?

**Comment by usmanrana07** [comment]
Yes, During swipe it may miss touch at any random images. Not necessarily on same images all the time.
